async-query-core/src/main/java/org/opensearch/sql/spark/client/EMRServerlessClientFactory.java

```diff
@@ -11,7 +11,8 @@ public interface EMRServerlessClientFactory {
   /**
    * Gets an instance of {@link EMRServerlessClient}.
    *
+   * @param accountId Account ID of the requester. It will be used to decide the cluster.
    * @return An {@link EMRServerlessClient} instance.
    */
-  EMRServerlessClient getClient();
+  EMRServerlessClient getClient(String accountId);
```

Comment:
This seems odd passing a parameter without using it. Can we make existing code use accountId?

Reply:
It is not needed for open source use case, since we won't have multiple EMRS cluster.